shared/components/EventSchedule/index.scss

```diff
@@ -1,8 +1,15 @@
 .EventSchedule {
+  border-right: none;
```

Comment:
is this needed? As this is mobile first I can't see why we'd need to override line 13